packages/plugin-core/src/commands/pods/ExportPodV2Command.ts

```diff
@@ -70,4 +70,11 @@ export class ExportPodV2Command extends BaseCommand<
   async execute(opts: CommandOpts) {
     opts.run();
   }
+
+  addAnalyticsPayload(opts: CommandOpts) {
+    return {
+      configured: true,
+      pod: opts.key,
+    };
+  }
```

Comment:
using the export pod will result in two tracked events (`dendron.exportpodV2` and `dendron.markdownexportv2`)

Should these two payloads be combined? the property `pod` is basically implied by the second tracked event

Reply:
Yeah, you are right, each export pod results in tracking both the export pod and the selected pod-specific command. This just adds an option to filter metrics based on export pod command and not go through individual ones. We can totally skip this if it does not make sense.